test/library/standard/DataFrames/DataFrames.chpl

```diff
@@ -550,32 +549,35 @@ module DataFrames {
     }
   }
 
+  // TODO: isNumericType prevents instantiation with bools
+  // would prefer "is summable" type here
```

Comment:
I'm somewhat out of the loop, but would it be worth defining `isSummableType(type t)` in `DataFrames` for now?

Reply:
I don't think so, since at present that would just be `isNumeric` and `isString`. What I meant is that classes should be able to implement a concept such that they are `Summable`.